pkg/util/containers/v2/metrics/collector_docker_windows.go

```diff
@@ -0,0 +1,59 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build docker && windows
+// +build docker,windows
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util"
+	"github.com/docker/docker/api/types"
+)
+
+func convertContainerStats(stats *types.Stats) *ContainerStats {
+	return &ContainerStats{
+		Timestamp: time.Now(),
+		CPU:       convertCPUStats(&stats.CPUStats),
+		Memory:    convertMemoryStats(&stats.MemoryStats),
+		IO:        convertIOStats(&stats.StorageStats),
+		PID:       convertPIDStats(stats.NumProcs),
+	}
+}
+
+func convertCPUStats(cpuStats *types.CPUStats) *ContainerCPUStats {
+	return &ContainerCPUStats{
+		Total:  util.Float64Ptr(float64(100 * cpuStats.CPUUsage.TotalUsage)),
```

Comment:
I think it'd be nice to add a comment that explains why `100 *` is needed (what are the units of the usage in Windows).

Reply:
Fixed: https://github.com/DataDog/datadog-agent/pull/9612/commits/32bc1af90ab330d45edb7591ef0458b24a1c0eed#diff-ff1cf6879822b2f78e4d27ad3529ed3bd61f2a6dc12c5819baf1145a1a757f6fR32